python_src/src/lamp_py/ingestion/convert_gtfs_rt.py

```diff
@@ -311,28 +284,44 @@ def gz_to_pyarrow(
                 ) as file:
                     json_data = json.load(file)
 
-            # Create empty 'table' as dict of lists for export schema
-            table = self.detail.empty_table()
-
             # parse timestamp info out of the header
             feed_timestamp = json_data["header"]["timestamp"]
             timestamp = datetime.fromtimestamp(feed_timestamp, timezone.utc)
 
-            # for each entity in the list, create a record, add it to the table
-            for entity in json_data["entity"]:
-                record = self.record_from_entity(entity=entity)
-                record.update(
-                    {
-                        "year": timestamp.year,
-                        "month": timestamp.month,
-                        "day": timestamp.day,
-                        "hour": timestamp.hour,
-                        "feed_timestamp": feed_timestamp,
-                    }
-                )
+            table = pyarrow.Table.from_pylist(
+                json_data["entity"], schema=self.detail.import_schema
+            )
```

Comment:
`from_pylist` creates pyarrow table from python list of objects. defining a schema will automatically insert `null` values for any schema fields not found in list of objects.